_data/banking.yml

```diff
@@ -6,6 +6,17 @@ websites:
       img: actorsfcu.png
       tfa: No
 
+    - name: Addiko Bank
+      url: https://www.addiko.hr/
+      img: addikobank.png
+      tfa: Yes
+      sms: No
+      phone: No
+      hardware: Yes
+      software: Yes
+      doc: https://www.addiko.hr/sites/addiko.hr/files/content/addiko_bank_uputa_za_koristenje_tokena_mtokena.pdf
+      lang: hr
```

Comment:
`lang: hr` can be removed because it should not be used when `tfa` is set to `Yes`.